rpm/cray/csm/noos/index.yaml

```diff
@@ -23,20 +23,40 @@
 #
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/noos/:
   rpms:
-    - acpid-2.0.31-2.1.x86_64.rpm
+    - acpid-2.0.31-2.1.aarch64
+    - acpid-2.0.31-2.1.x86_64
+    - bos-reporter-2.6.0-1.noarch
     - cani-0.1.0-1.x86_64
+    - canu-1.7.5-1.x86_64
+    - cray-kubectl-hns-plugin-1.0.2-1.x86_64
+    - cray-kubectl-kubelogin-plugin-1.25.3-1.x86_64
+    - cray-prodmgr-1.1.6-1.x86_64
     - cray-site-init-1.32.0-1.x86_64
+    - cray-uai-util-2.2.1-1.noarch
+    - craycli-0.82.9-1.aarch64
+    - craycli-0.82.9-1.x86_64
     - csm-auth-utils-1.0.0-1.noarch
-    - csm-node-heartbeat-2.0-3.x86_64.rpm
+    - csm-node-heartbeat-2.1-3.aarch64
+    - csm-node-heartbeat-2.1-3.x86_64
+    - csm-node-identity-1.0.22-1.noarch
+    - csm-testing-1.16.51-1.noarch
+    - dracut-metal-dmk8s-2.1.1-1.noarch
+    - dracut-metal-luksetcd-2.1.1-1.noarch
+    - dracut-metal-mdsquash-2.4.3-1.noarch
```

Comment:
There was some overlap between these manifests and the CSM embedded repo before, but now there is more.

The `/^dracut-/` packages don't need to exist here, it'll be more work to update them.

I believe the same goes for `/^cray-kubectl-*/`.

Basically, a lot of the _new_ packages this PR is added should be in the embedded repository already. The list of packages in these manifests should really be:
- Packages that we install outside of CFS
- Packages belonging to images that aren't used for constructing the embedded repository (e.g. application, csm compute, cos compute, and aarch64 flavors)

Perhaps @mtupitsyn has some insight, or can back me up. Going off of what I said, there should already have been items we could've removed.

Reply:
I guess I'm not clear on what the embedded repo is versus these. I thought that if CFS was going to install an RPM, it got it from these repos in Nexus. Is that not right?

Reply:
"Embedded repo" is a collection of RPMs installed in CSM tarball and Nexus. List of RPMs is taken from NCN node images. I.e. for almost every RPM pre-installed on NCN image, there's no need to add it here, because it will be in nexus anyway.

List of RPMs for embedded repo is printed during `Verify RPM Index` phase of csm build. Builds on this PR are all failing before list is printed, but you can see the list for unmodifie `release/1.5` branch here (click on "RPM index" stage): https://jenkins.algol60.net/blue/organizations/jenkins/Cray-HPE%2Fcsm/detail/release%2F1.5/31/pipeline/35

Reply:
As @rustydb mentioned above, `dracut` and `cray-kubectl` is already in the list, no need to add them under `rpm/`.

Reply:
However, please note, that in embedded repo, we pull packages by their name, by configuring repos with `${basearch}` zypper variable with `${NCN_ARCH}` environment variable from `assets.sh`. I.e. we only configure `x86_64` repos at the moment. Which means we pull `x86_64` and `noarch` packages, but not `aarch64`. Let me know if you want to change this logic (for example, automatically try to pull `aarch64` version of every `x86_64` package into embedded repo).

Reply:
I discussed this with Rusty. I understand the reasoning and will remove the packages I added based on my incomplete understanding of the embedded repo